Improve readability in the docs for Popen capture_output (#115581)
python/cpython: Doc/library/subprocess.rst

   this function are passed through to that interface. (*timeout*,  *input*,
   *check*, and *capture_output* are not.)

>    If *capture_output* is true, stdout and stderr will be captured. In this

Reviewer: ```suggestion
   If *capture_output* is true, stdout and stderr will be captured, and
```

>    scenario, the internal :class:`Popen` object is automatically created with

Reviewer: ```suggestion
   the internal :class:`Popen` object is automatically created with
```

>    ``stdout=PIPE`` and ``stderr=PIPE``. You must not specify any values for

Reviewer: ```suggestion
   ``stdout=PIPE`` and ``stderr=PIPE``. The *stdout* and *stderr* arguments may
   not be supplied at the same time as *capture_output*.
```

>    *stdout* and *stderr* arguments in this case. If you wish to capture

Reviewer: ```suggestion
   If you wish to capture
```
